src/Map/src/Bridge/Google/src/Option/MapTypeId.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Google\Option;
+
+/**
+ * Identifiers for common MapTypes.
```

Comment:
If there are other MapTypes this should not be an enum ;)

Reply:
There is only 4 MapTypeId https://developers.google.com/maps/documentation/javascript/reference/3.56/map?hl=en#MapTypeStyle

Reply:
As constant of their lib yes, but many more values  exist (they have a registry for it)

Reply:
Hum, you're right, didn't know about https://developers.google.com/maps/documentation/javascript/reference/3.56/map?hl=en#MapTypeRegistry

Reply:
DOne